ScannerScaffolding/ScannerScaffolding.go

```diff
@@ -150,18 +148,50 @@ func (scanner ScannerScaffolding) fetchJob() *ScanJob {
 	status := strings.Trim(res.Status, " ")
 
 	switch status {
+	case "200":
+		return scanner.parseFetchJobResponse(res)
 	case "204":
 		log.Debug("No jobs available. Going to sleep.")
 		scanner.logSuccessfulEngineConnection()
 		return nil
 	case "400":
 		log.Warning("Invalid Response / Request to engine while fetching a new job.")
 		return nil
+	case "401":
+		log.Warning("User not authorized. Did you set the environment variables to authenticate to the engine?")
+		return nil
 	case "500":
 		log.Warning("Encountered 500 Response Code from Engine while fetching a new job.")
 		return nil
+	default:
+		log.Warningf("Unexpected response status code '%s'", status)
+		return nil
 	}
+}
 
+func doHTTPRequest(method, url string, body io.Reader) (retres *http.Response, reterr error) {
```

Comment:
Typo in “retres” and “reterr” 😉

Reply:
Yes. Since "error" and "Response" already is pretty self-explanatory, I removed the return parameter names completely.